server/util.es6

```diff
@@ -1,8 +1,12 @@
 'use strict';
 
+let AWS = require('aws-sdk');
+let fs = require('fs');
 let moment = require('moment');
 let NoncommercialApplication = require('./models/noncommercial-application.es6');
+let request = require('request');
 let TempOutfitterApplication = require('./models/tempoutfitter-application.es6');
+let vcapServices = require('./vcap-services.es6');
 
 let extractField = (errorObj, withArg) => {
   if (withArg && errorObj.property === 'instance') {
```

Comment:
Seems like the code below is now duplicative of what is in the individual permit types.